### DISCUSSION TITLE
Catch exceptions before creating files

### DISCUSSION THREAD

**Discussion by muath-ye** [livewire/livewire#5152] [3. Feature ideas]
I run the following command to create new component:

```bat
php artisan make:livewire inputs.string
```

And I got the files created successfully but ```String``` class is reserved as it is php keyword, So I need to update the class name and the view name manually or remove them and re-run the command to create the component with different name.

![image](https://user-images.githubusercontent.com/34031333/182326552-a09482cf-863e-4f76-ab6e-18ce267bcdb4.png)

**Comment by joshhanley** [comment]
@muath-ye you should be able to use the move command to rename everything, see here https://laravel-livewire.com/docs/2.x/artisan-commands#move-command

Not sure how hard it would be to detect reserved workds or not, but PR's are welcome! 🙂
